### DISCUSSION TITLE
what should I do to handle about the table column name as same with table name when generate diesel schama

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [diesel-rs/diesel#3000] [Q&A]
I am using diesel cli to generate a schema, but when the table column name as same with table name, the compile will throw eorror:

error: Column `article_content` cannot be named the same as its table.

I was wondering if someone knows how to handle this situtation properly, I have no idea about how to handle this error. any suggestion is welcome. this is what I am facing right now:https://stackoverflow.com/questions/70438471/error-column-article-content-cannot-be-named-the-same-as-its-table

Or there is noway to handle this situation right now that I must change my table name or column name to avoid this problem? This is the full output of the error:

```
   Compiling reddwarf-admin v0.1.0 (/Users/xiaoqiangjiang/source/reddwarf/backend/reddwarf-admin)
error: Column `article_content` cannot be named the same as its table.
 You may use `#[sql_name = "article_content"]` to reference the table's `article_content` column.
 See the documentation of the `table!` macro for details`

  --> src/model/diesel/dolphin/dolphin_schema.rs:52:1
   |
52 | / table! {
53 | |     article_content (id) {
54 | |         id -> Int8,
55 | |         article_id -> Int8,
56 | |         article_content -> Varchar,
57 | |     }
58 | | }
   | |_^
   |
   = note: this error originates in the macro `__static_cond` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0609]: no field `articleContent` on type `ArticleContent`
  --> src/model/response/app/cruise/article/article_response.rs:40:38
   |
40 |             content: article_content.articleContent
   |                                      ^^^^^^^^^^^^^^ help: a field with a similar name exists: `article_content`

For more information about this error, try `rustc --explain E0609`.
error: could not compile `reddwarf-admin` due to 2 previous errors

```

**Comment by weiznich** [comment]
Generally speaking: 

* Please always include the complete error message as emitted by the compiler. In this case it does contain a quite explicit hint how to solve the problem.
* Asking the same question again and again will not give you a different answer. I really do not want to repeat the answer again and again.
* The "idea" category is for suggesting new features. That implies that you write down a proposal on how to improve a certain part of the API. It's not for asking questions.

**Reply by jiangxiaoqiang** [reply]
I pasted the full error message associate with this problem. I still facing this problem, any suggestion is welcome. I really want to solve this problem or get to know is there any chance to solve it.
